x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/logs_ui_categories/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "logs_ui_categories",
+  "title": "Log entry categories",
+  "description": "Categorize log entries in the Logs UI",
```

Comment:
This description will be shown in the anomaly detection job wizard in ML, so I think we should have 'detect' and / or 'anomalies' in there somewhere. Something like 'Detect anomalies in log entry categories' or 'Detect unusual counts of log entries by category'?